### ISSUE TITLE
Increased prealbumin level

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5944]
**Preferred term label:** Increased prealbumin level

**Synonyms**

**Definition (free text, please give PubMed ID)** An increased concentration of prealbumin in the blood., PMID:31094111

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added, also decreased
